pom.xml

```
@@ -63,7 +63,7 @@
   </scm>
 
   <properties>
-    <jenkins.version>2.164.2</jenkins.version>
+    <jenkins.version>2.263</jenkins.version>
```

Could we update this to the LTS `2.263.4`?

```suggestion
    <jenkins.version>2.263.4</jenkins.version>
```